docs/pages/setup/security/hardened-container-images.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Hardened Container Images
+description: How to run Teleport in hardened container images
+---
+
+Customers running in heavily regulated or compliance environments may be
+interested in running Teleport within a minimal or hardened container image.
+This may be customers running in FedRAMP environments or customers that have a
+requirement to meet CIS Hardened Image guidelines.
+
+We have plans to publish harded container images in the second half of 2022.
+
+In the meantime, customers can construct a hardened image that meets those
+requirement by running Teleport within `distroless` container image using the
```

Comment:
While distroless may add security due to a reduced surface area, I'm not sure it will actually help meet the requirements you reference here (FedRAMP or CIS). Typically these require you to meet rules for which hardening guidelines are available for, and the orgs that develop these guidelines only target the major OS images.

For example:
- CIS provides a hardened container image that is based on Ubuntu 16.04. 
- DISA has STIGs for [RHEL](https://www.stigviewer.com/stig/red_hat_enterprise_linux_7/) and [Ubuntu](https://ubuntu.com/security/disa-stig)